php/async/ExchangeCommon.php

```diff
@@ -0,0 +1,295 @@
+<?php
```

Comment:
Added generated files here just so you can see how it looks like without building it